felles/src/main/kotlin/no/nav/familie/kontrakter/felles/simulering/Simuleringsoppsummering.kt

```diff
@@ -21,5 +23,25 @@ data class Simuleringsperiode(
         val nyttBeløp: BigDecimal,
         val tidligereUtbetalt: BigDecimal,
         val resultat: BigDecimal,
-        val feilutbetaling: BigDecimal,
-)
+        val feilutbetaling: BigDecimal
+) {
+    constructor(
+            periode: Periode,
+            forfallsdato: LocalDate,
+            nyttBeløp: BigDecimal,
+            tidligereUtbetalt: BigDecimal,
+            resultat: BigDecimal,
+            feilutbetaling: BigDecimal
+    ) : this(
+          periode.fom,
+          periode.tom,
+          forfallsdato,
+          nyttBeløp,
+          tidligereUtbetalt,
+          resultat,
+          feilutbetaling
+    )
+
+    val periode: Periode
+        get() = Periode(fom,tom)
+}
```

Comment:
Jeg synes primærkonstruktøren er den som burde inneholde periode, så kan du heller lage gettere, val fom: LocalDate   get() = periode.fom og val tom: LocalDate get() = periode.tom. Da for blir feltene du henter de samme som er på objektet og du trenneger ikke lage nye periode-instanaser hele tiden.

Reply:
Jeg er enig. Men prøvde ikke å endre kontrakten for mye.